### PR TITLE
EVG-19083 Allow clearing all test filters and table refactors

### PR DESCRIPTION
**Pull request by khelif96**
EVG-19083

### Description
* Refactors the table controls to make them more reusable
* Adds a clear all button to the tests table
* Refactors pagination component and removes antd variant. 
* * This seemed like a natural step to move away from antd and make this component simpler to use (The [leafygreen pagination component](https://www.mongodb.design/component/pagination/example/) was an option but it seems to not be independent enough and probably makes sense to introduce once we fully migrate off of the antd tables)
* Add table controls to bottom of all tables for consistency
* 
### Screenshots
image

### CODE REVIEW

File: cypress/integration/version/task_table.ts [hunk 1]
```diff
@@ -80,7 +80,7 @@ describe("Task table", () => {
 
   ["NAME", "STATUS", "BASE_STATUS", "VARIANT"].forEach((sortBy) => {
     // TODO: This test doesn't work bc of issues with assertCorrectRequestVariables
```
Comment (minnakt) [inline cypress/integration/version/task_table.ts:82]: Could you open a ticket for this if we don't already have one? Seems like it's from three years ago back when Spruce was on Cypress 5, and I would guess we might be able to fix it now. Alternatively if it seems unimportant just delete it 🤷

File: src/pages/host/HostTable.tsx [hunk 2]
```diff
@@ -60,9 +60,8 @@ export const HostTable: React.VFC<{
         <PaginationWrapper>
           <Pagination
             data-cy="host-event-table-pagination"
-            pageSize={limit}
-            value={page}
-            totalResults={eventsCount}
+            currentPage={page}
```
Comment (minnakt) [inline src/pages/host/HostTable.tsx:63]: seeing some weird styles on event log pages!

Screenshot 2023-05-22 at 1 14 49 PM

**Review by minnakt — CHANGES_REQUESTED** [review]

File: cypress/integration/version/task_table.ts (hunk 1 above)
Comment (khelif96) [reply cypress/integration/version/task_table.ts:82]: Better yet I will include this in the table migration project. https://jira.mongodb.org/browse/EVG-19972

File: src/components/Table/TableWrapper.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import styled from "@emotion/styled";
+import { size } from "constants/tokens";
+
+interface TableWrapperProps {
+  children: React.ReactNode;
+  controls?: React.ReactNode;
+  shouldShowBottomTableControl?: boolean;
+}
+const TableWrapper: React.FC<TableWrapperProps> = ({
```
Comment (minnakt) [inline src/components/Table/TableWrapper.tsx:9]: this is awesome!! 😳

**Review by minnakt — APPROVED** [review]
looks great! 🤩

**Pull request merged**